### DISCUSSION TITLE
Inventor hub battery not charging

### DISCUSSION THREAD

**Discussion by Cervesi** [pybricks/support#723] [Q&A] [answered]
Hello, my inventor hub running pybricks does not charge the battery anymore.

If I connect the hub to power with USB:
-The 9 central LEDs turn on, the button flashes blue then orange, then after 2-3 seconds everything shuts down
-After, it emits an audible buzzing sound
-The red light next to the plug stays on

If I try to enter DFU mode:
-The bluetooth light flashes pink/green/blue
-The hub however is not detected, zadig does not help

In this moment the battery is empty and I cannot operate the hub. What alternatives do I have? Is there a way to charge the battery without a hub?

Thanks
Davide

**Comment by dlech** [comment]
> After, it emits an audible buzzing sound

This sounds like there could be a hardware problem. I would suggest contacting LEGO customer support to see what they say about it.

> The red light next to the plug stays on

This means that the battery should still be charging. I would normally suggest to leave it plugged in for an hour like this to see if the battery charges, but the fact that it is making a noise during this time is concerning.

**Comment by laurensvalk** [comment, marked as answer]
The inventor hub can do a DFU update even without the battery plugged in. The only thing is that you won't see the Bluetooth light blink.

How did you install Pybricks? Via DFU?

A different charger type and cable may help as well. I think I've seen this once before, with the stock firmware.

I was either charging from a laptop and then switched to an external phone charger (or the other way around, it has been a while), and then it started working again after charging for an hour or two.

**Reply by Cervesi** [reply]
> A different charger type and cable may help as well. I think I've seen this once before, with the stock firmware.

Thank you, that was it. I tried a couple different cables and found one that works. Apparently my usual charging station was tampered with without me knowing. Thanks for the help!
